item-impl/src/main/java/com/example/auction/item/impl/ItemRepository.java

```diff
@@ -182,15 +186,15 @@ public PItemEventProcessor(CassandraSession session, CassandraReadSide readSide)
         private CompletionStage<Done> prepareInsertItemSummaryByCreatorStatement() {
             return session.
                     prepare("INSERT INTO itemSummaryByCreator(" +
-                            "creatorId, " +
                             "itemId, " +
+                            "creatorId, " +
                             "title, " +
```

Comment:
I put `creatorId` first in most of the things dealing with `itemSummaryByCreator`, because I wanted that to be the partition key.

It doesn't _really_ matter for the insert statement, but it was originally put in that order for consistency with the table definition. I don't mind changing it, but just wanted to point out that it wasn't an accident.